app/Domain/Integrations/Environment.php

```diff
@@ -9,4 +9,5 @@ enum Environment: string
     case Acceptance = 'acc';
     case Testing = 'test';
     case Production = 'prod';
+    case Unknown = 'unknown';
```

Comment:
Why do we need this extra environment?

Reply:
Removed it. The only downside is that the Master realm is now attached to the production realm, which is not entirely correct.
But the only place we use the Environment is to match the correct IntegrationUri to the correct realm, so it is not a problem.